services/navigation/builtin/builtin.go

```diff
@@ -19,13 +19,25 @@ import (
 	"go.viam.com/rdk/resource"
 	"go.viam.com/rdk/services/motion"
 	"go.viam.com/rdk/services/navigation"
+	"go.viam.com/rdk/services/vision"
 	"go.viam.com/rdk/spatialmath"
 	rdkutils "go.viam.com/rdk/utils"
 )
 
 const (
-	metersPerSecDefault = 0.5
-	degPerSecDefault    = 45
+	defaultMetersPerSec = 0.5
+	defaultDegsPerSec   = 45
+
+	// how far off the path must the robot be to trigger replanning.
+	defaultPlanDeviationM = 1e9
+
+	// the allowable quality change between the new plan and the remainder
+	// of the original plan.
+	replanCostFactorDefault = 1.
```

Comment:
`defaultReplanCostFactor` to be consistent

Reply:
ai yay yaiy - im sorry about this. I should have been more thorough